service/systemd/service.go

```diff
@@ -230,8 +237,18 @@ func (s *Service) readConf() (common.Conf, error) {
 	var conf common.Conf
 
 	data, err := Cmdline{}.conf(s.Service.Name, s.DirName)
-	if err != nil {
+	errString := fmt.Sprintf("%v", err)
+	if err != nil && strings.Contains(errString, "No such file or directory") == false {
```

Comment:
Rather than `strings.Contains(errString, "No such file or directory") == false`, this should be `!strings.Contains(errString, "No such file or directory")`.